ext/node/analyze.rs

```diff
@@ -111,67 +113,88 @@ impl<TCjsCodeAnalyzer: CjsCodeAnalyzer> NodeCodeTranslator<TCjsCodeAnalyzer> {
       .map(|s| s.to_string())
       .collect::<HashSet<_>>();
 
-    // (request, referrer)
-    let mut reexports_to_handle = VecDeque::new();
-    for reexport in analysis.reexports {
-      reexports_to_handle.push_back((reexport, specifier.clone()));
-    }
-
-    // todo(dsherret): we could run this analysis concurrently in a FuturesOrdered
-    while let Some((reexport, referrer)) = reexports_to_handle.pop_front() {
-      // First, resolve the reexport specifier
-      let reexport_specifier = self.resolve(
-        &reexport,
-        &referrer,
-        // FIXME(bartlomieju): check if these conditions are okay, probably
-        // should be `deno-require`, because `deno` is already used in `esm_resolver.rs`
-        &["deno", "require", "default"],
-        NodeResolutionMode::Execution,
-        permissions,
-      )?;
-
-      if !handled_reexports.insert(reexport_specifier.clone()) {
-        continue;
-      }
+    let mut handled_reexports: HashSet<ModuleSpecifier> = HashSet::default();
+    handled_reexports.insert(entry_specifier.clone());
+    let (reexports_to_handle_tx, mut reexports_to_handle_rx) =
+      deno_core::unsync::mpsc::unbounded_channel();
+    reexports_to_handle_tx
+      .send((entry_specifier.clone(), analysis.reexports))
+      .unwrap();
+    let mut analyze_futures: FuturesUnordered<
+      LocalBoxFuture<Result<_, AnyError>>,
+    > = FuturesUnordered::new();
```

Comment:
We should create an unsync version of this (not joinset, which will spawn futures)

Reply:
https://github.com/denoland/deno_unsync/pull/9